test/fixtures/v5/end_of_road/left_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "end of road",
+            "modifier": "left"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
```

Comment:
Not sure when end-of-road can occur in combination with an exit - these may be good candidates for validation handlers? cc @MoKob